driver/config/config.go

```diff
@@ -527,10 +539,37 @@ func (p *Config) SecretsSession() [][]byte {
 	return result
 }
 
+func (p *Config) SecretsAES() (secret string) {
+	secret = p.p.String(ViperKeySecretsAES)
+	if len(secret) == 0 {
+		secret = string(p.SecretsDefault()[0][:32])
+	}
+	if len(secret)%32 != 0 {
+		p.l.Warnf("secret bad size %d, generate a key", len(secret))
+		secret = p.generateKey()
+	}
+	return
+}
+
 func (p *Config) SelfServiceBrowserDefaultReturnTo() *url.URL {
 	return p.ParseURIOrFail(ViperKeySelfServiceBrowserDefaultReturnTo)
 }
 
+func (p *Config) generateKey() string {
+	const letters = "0123456789abcdef"
+	const length = 128
+	ret := make([]byte, length)
+	for i := 0; i < length; i++ {
+		num, err := rand.Int(rand.Reader, big.NewInt(int64(len(letters))))
+		if err != nil {
+			p.l.WithError(err).Error("generate random key failed")
+			return letters + letters
+		}
+		ret[i] = letters[num.Int64()]
+	}
+	return string(ret)
+}
```

Comment:
Please use a crypto byte reader and pipe it through, for example, sha256, to generate strong keys.

Reply:
use the key generator from cryptopasta